.ci/test/static/run

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script runs static tests which
+# can be performed without code execution.
+#
+# Some example are code sniffing and unit testing
+#
+
+# Run composer install to get the vendor directory
+composer install --no-ansi --no-interaction --optimize-autoloader --no-progress
+
+# Delete and reset the web directory
+# This is necessary so that third-party
+# dependencies are not tested
+rm -rf web
+git checkout -- web
+
+# Lint php code for syntax errors
+composer -n lint
+
+# Check coding standards
+composer -n code-sniff
+
+# Run unit tests
+composer -n unit-test
```

Comment:
The unit test might fail after cleaning the `web` directory.  We could lint / sniff without running `composer install`, but running the unit tests will likely need vendored code, so I suppose there's little point in avoiding the install.

The efficient way to resolve this would be to re-order the tests so that the unit tests ran before cleaning the web directory. That's probably better than running `composer install` twice, even though we might otherwise prefer to lint and sniff first.

Reply:
Updated in `54ad541439ac6d0f8886a9c51efe4c3d119bcb66`

Reply:
This should probably be ported to `example-wordpress-composer` as well